### PR TITLE
PR for authentication screens

### PR DESCRIPTION
**Pull request by adidoshi**
### Hey I created the authentication pages for e-commerce! Please review.
* Sign-in screen -> [Live preview](https://deploy-preview-3--splash-shop.netlify.app/screens/auth/login)
* Sign-up screen -> [Live preview](https://deploy-preview-3--splash-shop.netlify.app/screens/auth/register)
* Forgot-password screen -> [Live preview](https://deploy-preview-3--splash-shop.netlify.app/screens/auth/forgotpassword)

### CODE REVIEW

**Review by srishti-maurya — APPROVED** [review]
looks good

**Review by tanveertkd — APPROVED** [review]
Looks good to go!

File: screens/auth/Login.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+            ></span>
+            <div class="error-message">Incorrect Password</div>
+          </div>
+          <div class="login-form-forgot">
+            <a href="./ForgotPassword.html">Forgot Password?</a>
+          </div>
+          <button class="card-actions-btn-filled">Login</button>
+
+          <a href="./Register.html" class="form-link"
+            >New here, create an account?</a
```
Comment (MarkVed17) [inline screens/auth/Login.html]: New here? Create an Account*
This is better imo.

File: screens/auth/Form.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.login-form {
+  box-shadow: var(--card-box-shadow);
+  height: max-content;
+  padding: 1.6rem;
+  width: 375px;
```
Comment (MarkVed17) [inline screens/auth/Form.css]: using rem consistently is a good practice & must be followed. You may do these changes later though.

**Review by MarkVed17 — APPROVED** [review]
UI looks good. Just a few good practices to be followed 👍🏼

File: screens/auth/Login.html (hunk 1 above)
Comment (adidoshi) [reply screens/auth/Login.html]: Yes, I'll change that!

File: screens/auth/Form.css (hunk 2 above)
Comment (adidoshi) [reply screens/auth/Form.css]: Thanks! Surely I'll follow that.

**Pull request merged**